src/api-serverless/package.json

```diff
@@ -7,11 +7,8 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "dev": "NODE_ENV=local serverless offline",
     "prebuild": "rm -rf dist",
-    "build": "esbuild src/handler.ts --bundle --sourcemap --platform=node --target=es2020 --outfile=dist/index.js && mkdir dist/native_modules && cp -r node_modules/sharp dist/native_modules/sharp/ && mkdir dist/node_modules && cp -r node_modules/semver dist/node_modules/semver/ && cp -r node_modules/detect-libc dist/node_modules/detect-libc/ && cp -r node_modules/color dist/node_modules/color/ && cp -r node_modules/color-string dist/node_modules/color-string/ && cp -r node_modules/color-name dist/node_modules/color-name/ && cp -r node_modules/color-convert dist/node_modules/color-convert/ && cp -r node_modules/simple-swizzle dist/node_modules/simple-swizzle/ && cp -r node_modules/is-arrayish dist/node_modules/is-arrayish/",
-    "postbuild": "cd dist && zip -r index.zip *",
-    "prebuild:linux": "rm -rf dist && npm uninstall sharp && npm install --arch=x64 --platform=linux --libc=glibc sharp",
-    "build:linux": "esbuild src/handler.ts --bundle --sourcemap --platform=node --target=es2020 --outfile=dist/index.js && mkdir dist/native_modules && cp -r node_modules/sharp dist/native_modules/sharp/ && mkdir dist/node_modules && cp -r node_modules/semver dist/node_modules/semver/ && cp -r node_modules/detect-libc dist/node_modules/detect-libc/ && cp -r node_modules/color dist/node_modules/color/ && cp -r node_modules/color-string dist/node_modules/color-string/ && cp -r node_modules/color-name dist/node_modules/color-name/ && cp -r node_modules/color-convert dist/node_modules/color-convert/ && cp -r node_modules/simple-swizzle dist/node_modules/simple-swizzle/ && cp -r node_modules/is-arrayish dist/node_modules/is-arrayish/",
-    "postbuild:linux": "cd dist && zip -r index.zip * && cd .. && npm uninstall sharp && npm install sharp"
+    "build": "esbuild src/handler.ts --bundle --sourcemap --platform=node --target=es2020 --outfile=dist/index.js && mkdir dist/node_modules && cp -r node_modules/semver dist/node_modules/semver/ && cp -r node_modules/detect-libc dist/node_modules/detect-libc/ && cp -r node_modules/color dist/node_modules/color/ && cp -r node_modules/color-string dist/node_modules/color-string/ && cp -r node_modules/color-name dist/node_modules/color-name/ && cp -r node_modules/color-convert dist/node_modules/color-convert/ && cp -r node_modules/simple-swizzle dist/node_modules/simple-swizzle/ && cp -r node_modules/is-arrayish dist/node_modules/is-arrayish/",
```

Comment:
i think these were all sharp dependencies and should no longer be needed,

Reply:
Removed them all